eng/pipelines/templates/steps/analyze_dependency.yml

```diff
@@ -18,12 +18,6 @@ steps:
     parameters:
       ScanPath: ${{ parameters.ScanPath }}
 
-  - pwsh: |
-      mkdir "$(Build.ArtifactStagingDirectory)/reports"
-      Copy-Item -Path "$(Build.SourcesDirectory)/eng/common/InterdependencyGraph.html" -Destination "$(Build.ArtifactStagingDirectory)/reports/InterdependencyGraph.html"
```

Comment:
Was this just copying but not being used?

Reply:
100%. Relic.